pkg/kubernetes/kubernetes.js

```diff
@@ -194,16 +204,16 @@ define([
             var reqs = [];
 
             reqs.push(etcd_api.get("/v2/admin/machines")
-                .fail(failure)
```

Comment:
Why are all these lines changed? It seems perhaps they were accidentally indented here? If so, I would suggest reverting this change.